dpbench/benchmarks/pairwise_distance/pairwise_distance_dpnp.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2022 Intel Corp.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import dpnp
+
+def pairwise_distance(X1, X2, D):
+    x1 = dpnp.sum(dpnp.square(X1), axis=1)
+    x2 = dpnp.sum(dpnp.square(X2), axis=1)
+    D = -2 * dpnp.dot(X1, X2.T)
+    x3 = x1.reshape(x1.size, 1)
+    D += x3  # x1[:,None] Not supported by Numba
+    D += x2
+    D = dpnp.sqrt(D)
```

Comment:
This implementation will just update a copy of D which will not replace the original version of D. This is causing the validation to fail. Look at [this ](https://github.com/IntelPython/dpbench/blob/1107f8816de6ea131141850be9f29633d054cd27/dpbench/benchmarks/pairwise_distance/pairwise_distance_numba_np.py#L10)version that updates D in-place.

Reply:
@adarshyoga, Thanks for the proposal. Now it is fixed, however it was required to avoid "=" operation everywhere. Instead it is requried to use dpnp.add / dpnp.multiply / dpnp.copyto